src/pages/host/list.jsx

```diff
@@ -1,16 +1,36 @@
-import React from "react";
+import { Link, useLoaderData } from "react-router-dom";
+import { getVans } from "../../api";
+import "../../server";
+import { loader as loadLogOut } from "../authentication";
+
+export async function loader({ request }) {
+  return { vans: await getVans(), prevPath: loadLogOut({ request }) };
+}
 
 function List() {
-  return (
+  const list = useLoaderData();
+  console.log(list.prevPath);
+  const loading = false;
+
+  return loading ? (
+    <div className="text-center py-52">Loading...</div>
+  ) : (
```

Comment:
There is not reason to use the ternary operator as loading will always be false, so you can just return.

Reply:
DONE :heavy_check_mark: